extensions/amp-iframe/0.1/amp-iframe.js

```diff
@@ -281,13 +331,21 @@ export class AmpIframe extends AMP.BaseElement {
    * @private
    */
   activateIframe_() {
-    this.getVsync().mutate(() => {
-      if (this.placeholder_) {
-        this.iframe_.style.zIndex = '';
-        removeElement(this.placeholder_);
-        this.placeholder_ = null;
-      }
-    });
+    if (this.placeholder_) {
+      this.getVsync().mutate(() => {
+        if (this.iframe_) {
+          this.iframe_.style.zIndex = 0;
```

Comment:
I was getting an error on Travis saying "expected '0' to equal ''`, so I changed it. Maybe `auto` would be better here?